docs/siem/detections/rules-ui-create.asciidoc

```diff
@@ -1,95 +1,71 @@
 
 beta[]
 
-Rules run periodically and search for documents or {ml} job anomaly scores 
-that meet their criteria. For both prebuilt and custom rules, you can use the
-{kib} {kibana-ref}/alerting-getting-started.html[Alerting and Actions] feature
-to send notifications when signals are created. Notifications can be sent via
-Email, PagerDuty, Slack, and Webhook, and can be configured in the {siem-app}
-when you create or edit a rule.
+Rules run periodically and search for source events or {ml} job anomaly scores 
+that meet their criteria. When a rule's criteria are met, a detections alert is
+created.
 
-[IMPORTANT]
-==============
-After you activate a rule, periodically check the rule is running as expected
-in the <<monitor-rule-exe, Monitoring tab>> on the All rules page. If you see
-values in the `Gap` column, you can <<troubleshoot-signals>>.
+You can create these types of rules:
 
-When a rule fails to run, the {siem-app} tries to rerun it at its next 
-scheduled run time.
-==============
+* *Custom query*: Query-based rule, which searches the defined indices and creates an alert when a document matches the rule's query.
+* *Machine learning*: {ml-cap} rule, which creates an alert when a {ml} job discovers an anomaly above the defined threshold (see <<machine-learning>>).
++
+For {ml} rules, the associated {ml} job must be running. If the {ml} job is not
+running, the rule will:
 
-On the Rules page, you can:
+** Run and create alerts if existing anomaly scores above the defined threshold
+are discovered.
+** Issue an error stating the {ml} job was not running when the rule executed.
+* *Threshold rules*: Searches the defined indices and creates a detections alert
+when the number of times the specified field's value meets the threshold during
+a single execution. When multiple values meet the threshold, an alert is
+generated for each value.
++
+For example, if the threshold `field` is `source.ip` and its `value` is `10`, an
+alert is generated for every source IP address that appears in at least 10 of
+the rule's search results.
 
-* <<load-prebuilt-rules>>
-* <<select-all-prebuilt-rules>>
-* <<create-rule-ui, Create your own rules>>
-* <<manage-rules-ui, Clone, edit, delete, activate, and deactivate rules>>
-* <<monitor-rule-exe>>
-* <<import-export-rules-ui>>
-* <<troubleshoot-signals>>
+When creating or modifying rules, you can add exceptions that prevent a rule
+from generating an alert even when its criteria are met. This is useful for
+reducing noise, such as preventing alerts from trusted processes and internal
+IP addresses. <<detections-ui-exceptions>> describes how to add exceptions to a
+rule.
 
-[role="screenshot"]
-image::all-rules.png[]
-
-[float]
-[[load-prebuilt-rules]]
-=== Load prebuilt Elastic rules
+For both prebuilt and custom rules, you can use the
+{kib} {kibana-ref}/alerting-getting-started.html[Alerting and Actions] feature
+to send notifications when alerts are created. Notifications can be sent via
```

Comment:
> to send notifications **when alerts are created**. Notifications can be sent via

Perhaps this is called out in the actions section explicitly (in which case please ignore :), but notifications can be scheduled not just when alerts are created, but at pre-defined intervals as well (hourly/daily/weekly).

Reply:
This is mentioned below it looks like -- please do ignore :)